src/main/java/com/thevoxelbox/voxelsniper/brush/type/SnowConeBrush.java

```diff
@@ -1,18 +1,32 @@
 package com.thevoxelbox.voxelsniper.brush.type;
 
+import cloud.commandframework.annotations.CommandMethod;
+import cloud.commandframework.annotations.CommandPermission;
 import com.fastasyncworldedit.core.configuration.Caption;
 import com.fastasyncworldedit.core.registry.state.PropertyKey;
 import com.sk89q.worldedit.math.BlockVector3;
 import com.sk89q.worldedit.registry.state.Property;
 import com.sk89q.worldedit.world.block.BlockState;
 import com.sk89q.worldedit.world.block.BlockType;
 import com.sk89q.worldedit.world.block.BlockTypes;
+import com.thevoxelbox.voxelsniper.command.argument.annotation.RequireToolkit;
 import com.thevoxelbox.voxelsniper.sniper.snipe.Snipe;
 import com.thevoxelbox.voxelsniper.sniper.snipe.message.SnipeMessenger;
 import com.thevoxelbox.voxelsniper.util.material.Materials;
+import org.jetbrains.annotations.NotNull;
 
+@RequireToolkit
+@CommandMethod(value = "brush|b snow_cone|snowcone|snow")
+@CommandPermission("voxelsniper.brush.snowcone")
 public class SnowConeBrush extends AbstractBrush {
 
+    @CommandMethod("")
+    public void onBrush(
+            final @NotNull Snipe snipe
+    ) {
+        super.onBrushCommand(snipe);
+    }
+
     @Override
     public void handleArrowAction(Snipe snipe) {
```

Comment:
Is this wanted so that it only works with gunpowder? It doesn't seem to work on arrow item yet. I know that this might be not related to the migration.

Reply:
This behavior has been here for 10 years :shrug: _We could probably add the same behavior for the arrow, as well._